telemetry-impls/summarize/action.yml

```diff
@@ -105,4 +115,9 @@ runs:
                     --force-parent-span-id "$(cut -d'-' -f3 <<<"$job_traceparent")" \
                     --start "$(cat job_info.json | jq -r '.created_at // "now"')" \
                     --end "$(cat job_info.json  | jq -r '.started_at // "now"')" \
-                    --verbose
\ No newline at end of file
+                    --verbose
+
+    - name: always exit 0 so telemetry does not mask upstream errors
```

Comment:
could you elaborate on this? i'm not sure I understand

Reply:
This was Brad's idea. Github jumps to the last error in the stack. If there's an error further up AND in the telemetry summary, Github will show the telemetry error, but the other one will be collapsed. This was happening prior to https://github.com/rapidsai/shared-workflows/pull/261. The build error was real, and it prevented subsequent steps from running, except the telemetry step that has `if: always()`. The problem was that the skipped steps were needed for that telemetry step.

At any rate, another way to say this is:
* We always want telemetry, whether a job passes or fails
* Telemetry should never affect the pass/fail status of the overall job